### DISCUSSION TITLE
FR: option to disable the new feature 'Briefly show tab sidebar when switching tabs in compact mode'

### DISCUSSION THREAD

**Discussion by noidilin** [zen-browser/desktop#1389] [Ideas]
In release 1.0.0-a.39, a new feature has been introduced that "briefly shows the tab sidebar when switching tabs in compact mode." The sidebar temporarily appears when I switch tabs using `ctrl+tab` and `ctrl+1~9` or when I close a tab, as the browser automatically falls back to the last tab.

After using this feature for a few days, I found it somewhat distracting in my personal use case. Since the floating sidebar overlaps the web content, I often have to pause my reading to wait for the sidebar to disappear.

In previous releases, I consistently kept the compact mode setting enabled to maximize the viewing area for web content. I appreciated the floating nature of the compact mode sidebar, as it didn’t shift the entire browser layout like the standard mode did. I only activated the compact mode sidebar when I needed to navigate tabs more precisely, typically using the `ctrl+1~9` shortcut, and then turned it off once I was finished.

Additionally, I often switch between two or three newly opened tabs. I usually have a rough idea of the tab order, so using the `ctrl+tab` shortcut allows me to switch between them easily. While this may not be the most efficient way to browse through multiple tabs, it helps me maintain focus on my research. However, with the sidebar blocking part of the web content when I don’t need it, I find that my thought process is frequently interrupted.

Additionally, this feature causes the sidebar to appear even when the user has set it to be hidden. While it is undoubtedly useful feature for tab navigation, switching tabs doesn’t always mean the user wants to see the sidebar. In this case, the browser's behavior conflicts with the user's default setting which is keep the sidebar hidden.

In conclusion, if I could enable the sidebar only when I need to check the tab order, it would offer a more seamless experience. I wouldn't mind the sidebar temporarily overlapping the web content when it serves a purpose.

**Comment by mr-cheffy** [comment]
![image](https://github.com/user-attachments/assets/5f039b3d-61e7-4c63-9a2a-48e6833d9c5e)

**Reply by noidilin** [reply]
It's true that the sidebar won't block the content anymore, but I'd like to keep using the compact mode.

**Reply by n7itro** [reply]
> It's true that the sidebar won't block the content anymore, but I'd like to keep using the compact mode.

Just disable `Briefly make the toolbar popup when switching or opening new tabs in compact mode`

**Reply by noidilin** [reply]
I am so sorry for wasting everybody's time...
